pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -46,29 +49,22 @@ assert ngspiceSupport -> libngspice != null;
 
 with lib;
 let
-  versionConfig = versions.${baseName};
-
   # oce on aarch64 fails a test
   withOCE = oceSupport && !stdenv.isAarch64;
   withOCC = (withOCCT && !withOCE) || (oceSupport && stdenv.isAarch64);
 
-  libraries = callPackages ./libraries.nix versionConfig.libVersion;
-
 in
 stdenv.mkDerivation rec {
 
-  i18n = libraries.i18n;
+  i18n = callPackage ./i18n.nix {
+    src = i18nSrc;
+    version = i18nVersion;
+  };
```

Comment:
This is called twice eventually - not good. I think it fits more if it's in `default.nix` because it's the entry point of whoever that inspects this derivation.